packages/flutter/test/material/search_anchor_test.dart

```diff
@@ -1362,6 +1362,50 @@ void main() {
     expect(controller.value.text, suggestion);
   });
 
+  testWidgets('SearchAnchor suggestionsBuilder property could be async', (WidgetTester tester) async {
+    final SearchController controller = SearchController();
+    const String suggestion = 'suggestion text';
+
+    await tester.pumpWidget(MaterialApp(
+      home: StatefulBuilder(
+          builder: (BuildContext context, StateSetter setState) {
+            return Material(
+              child: Align(
+                alignment: Alignment.topCenter,
+                child: SearchAnchor(
+                  searchController: controller,
+                  builder: (BuildContext context, SearchController controller) {
+                    return const Icon(Icons.search);
+                  },
+                  suggestionsBuilder: (BuildContext context, SearchController controller) async {
+                    return <Widget>[
+                      ListTile(
+                          title: const Text(suggestion),
+                          onTap: () {
+                            setState(() {
+                              controller.closeView(suggestion);
+                            });
+                          }),
```

Comment:
These should be on separate lines, something like this:

```suggestion
                          },
                        ),
```

Reply:
Done.